### ISSUE TITLE
swift-stdlib-tool verbosity

### ISSUE THREAD

**Issue by hochhaus** [bazelbuild/bazel#2740]
### Description of the problem / feature request / question:

Currently swiftstdlibtoolwrapper [passes](https://github.com/bazelbuild/bazel/blob/master/src/tools/xcode/swiftstdlibtoolwrapper/swiftstdlibtoolwrapper.sh#L70) `--verbose` to `swift-stdlib-tool`. This results in extraneous logging during builds. Could `--verbose` no longer be on by default?

```sh
Copying libswiftCore.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftCoreGraphics.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftQuartzCore.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftFoundation.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftUIKit.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftDispatch.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftObjectiveC.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftCoreImage.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
Copying libswiftDarwin.dylib from /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator to /tmp/swiftstdlibtoolZippingOutput.cuLzoW/Frameworks
```

### If possible, provide a minimal example to reproduce the problem:

Any usage of `swift_library` will pull in the stdlib and generate this logging. For example:

```py
load("@bazel_tools//tools/build_defs/apple:swift.bzl", "swift_library")

swift_library(
    name = "slib",
    srcs = glob(["*.swift"]),
)

objc_binary(
    name = "bin",
    srcs = ["@bazel_tools//tools/objc:dummy.c"],
    deps = [":slib"],
)

ios_application(
    name = "app",
    binary = ":bin",
    infoplist = "Info.plist",
)
```

### Environment info
* Operating System: macOS 10.12.3
* Bazel version (output of `bazel info release`): release 0.4.5

**Comment by philwo** [comment]
@dmishe Could you have a look at this?

**Comment by dmishe** [comment]
Fixed in f1792e3a67a110d74eb134a93c6331aa0b61bf3c
